src/release_artifacts_resources/ios/cdk/app.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+from aws_cdk import core as cdk
+from cdk.s3_stack import S3Stack
+from cdk.cloudfront_stack import CloudFrontStack
+
+from aws_cdk import core
+
+from cdk.cdk_stack import CdkStack
+
+def raiseOperationValueError():
+    raise ValueError(
+        "Provide operation with value either 'CREDENTIAL_ROTATION' or 'RELEASE_HOST' in 'context' parameter, as in: cdk deploy app -c operation=CREDENTIAL_ROTATION"  # noqa: E501
+    )
+
+app = core.App()
+
+operation = app.node.try_get_context("operation")
+if operation is None:
+    raiseOperationValueError()
+
+main = CdkStack(app, "MainStack")
+
+if operation == 'RELEASE_HOST':
```

Comment:
I think this operation is intending to deploy the host/cloudfront infrastructure? "Release" is slightly ambiguous absent other context, because it could be interpreted to mean "release (clean up) the cloud resources". The other slight confusion is that "operation" sounds imperative, but we're actually setting up infrastructure. What about `stack` for the context parameter name and `DISTRIBUTION` and `CREDENTIAL_ROTATION` for the stack types?

```bash
% cdk deploy app -c stack=DISTRIBUTION
% cdk deploy app -c stack=CREDENTIAL_ROTATION -c bucket=my-s3-bucket-name distribution_id=mycloudfrontdistributionid
```

Another way to handle this would be to have separate stacks, which must be individually specified. By convention (maybe by default? I'm not sure) we would expect to do a `cdk deploy main...`, but we could set up 3 different stacks:
- **distribution**: The stack for setting up S3 and CloudFront
- **credential_rotation**: The stack for setting up the IAM user, policies, and Lambda rotator
- **main**: A no-op stack that would emit an error instructing users to specify one or the other of the `distribution` and `credential_rotation` stacks.

I'm not familiar enough with CDK idioms to know if that makes sense, so I don't have a strong opinion on which is the right way to isolate these.

Reply:
Liked the second method. I just created two stacks, because if you do a main stack that throws an error, it will always raise that error in `cdk ls`.  With the current change the `cdk ls` look like this:

```terminal

✗ cdk ls
CredentialRotationStack
DistributionStack

```